aspnetcore/data/ef-mvc/intro.md

```diff
@@ -102,13 +102,19 @@ Because you used the **Individual User Accounts** option when you created the pr
 
 If you want to add EF Core support to a new project that you create without the **Individual User Accounts** option, install the following NuGet packages:
 
-* The package for the database provider you want to target. To use SQL Server, the package is [Microsoft.EntityFrameworkCore.SqlServer](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.SqlServer/). For a list of available providers see [Database Providers](https://docs.microsoft.com/ef/core/providers/).
+* The package for the database provider that you want to target. For SQL Server, the package is [Microsoft.EntityFrameworkCore.SqlServer](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.SqlServer/). For a list of available providers see [Database Providers](https://docs.microsoft.com/ef/core/providers/).
+
+* The package for running EF command-line tools in **Package Manager Console**:[Microsoft.EntityFrameworkCore.Tools](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Tools). 
+
+* The package for running EF command-line tools at a command prompt: [Microsoft.EntityFrameworkCore.Tools.DotNet](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Tools.DotNet). 
 
-* The package for the EF command-line tools:   [Microsoft.EntityFrameworkCore.Tools](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Tools/1.0.0-preview2-final). This package is a preview release, so to install it you have to enable preview release installation. After installing the package, you also have to add a reference to it in the `tools` collection in the *project.json* file.
+  This package is installed by adding it to the `DotNetCliToolReference` collection in the *.csproj* file, as shown.
 
-If you open the *project.json* file, you'll see that these packages are already installed.
+  [!code-xml[](intro/samples/cu/ContosoUniversity.csproj?range=30-34&highlight=2)]
```

Comment:
We know customers will copy/paste and soon that version # will be out of date. But I think that's probably the least of the evils. Probably better to not have a note: These version numbers were correct at the time of writing. You generally want to download the latest stable version.